dspace/config/modules/uclouvain.cfg

```diff
@@ -34,6 +34,14 @@ event.consumer.authoritymetadataenhancer.filters = Item+All
 
 event.consumer.formfieldcleaner.class = org.dspace.uclouvain.consumer.FormFieldCleanerConsumer
 event.consumer.formfieldcleaner.filters = Item+All
+
+event.consumer.fillervalueremover.class = org.dspace.uclouvain.consumer.FillerValueRemoverConsumer
+event.consumer.fillervalueremover.filters = Item+Modify|Modify_Metadata
+event.consumer.fillervalueremover.fillers = NA
+event.consumer.fillervalueremover.fillers = na
+event.consumer.fillervalueremover.fillers = N/A
+event.consumer.fillervalueremover.fillers = n/a
```

Comment:
use a regexp to be more flexible --> `"\\s*(?i)(n/?a|not specified|no(t|n) applicable)\\s*"`

`\\s*` : n'importe quel nombre de d'espaces
`(?i)` : insensible à la case